### ISSUE TITLE
ReCAPTCHA V2 problems

### ISSUE THREAD

**Issue by fedelauberer** [ultrafunkamsterdam/undetected-chromedriver#169]
I had been using uc for months and had no problems with v2 captcha, but I've been trying today and it seems as if V2 captcha instantly marks me down as a bot. Was wondering if this is just happening to me or if something else changed recently?

**Comment by Nayeth2019** [comment]
I also cant log into gmail from within undetected chromedriver

**Comment by itopaulojr** [comment]
I'm looking for a way to bypass google login recaptcha. It seems that this lib is not working for this purpose, write?

**Comment by soermejo** [comment]
im not getting recaptcha 3 enterprise with this library

**Comment by julionc** [comment]
> I'm looking for a way to bypass google login recaptcha. It seems that this lib is not working for this purpose, right?

Nope. You can use a external service like 2Captcha to resolve them.
Close this
